app/handlers/karma.py

```diff
@@ -20,22 +20,43 @@
 
 @router.message(Command("top", prefix="!"), F.chat.type == "private")
 async def get_top_from_private(
-    message: types.Message, user: User, chat_repo: ChatRepo, user_repo: UserRepo
+    message: types.Message,
+    user: User,
+    chat_repo: ChatRepo,
+    user_repo: UserRepo,
+    config: Config,
+    bot: Bot,
 ):
     parts = message.text.split(maxsplit=1)
     if len(parts) == 1:
-        return await message.reply(
+        bot_reply = await message.reply(
             "Эту команду можно использовать только в группах "
             "или с указанием ID нужного чата, например:"
             "\n" + hd.code("!top -1001399056118")
         )
+        return asyncio.create_task(
+            cleanup_command_dialog(
+                bot=bot,
+                bot_message=bot_reply,
+                delete_bot_reply=True,
+                delay=config.time_to_remove_temp_messages,
+            )
+        )
     try:
         chat = await chat_repo.get_by_id(chat_id=int(parts[1]))
     except DoesNotExist:
-        return await message.reply(
+        bot_reply = await message.reply(
             "Не удалось найти чат с таким ID, убедитесь, "
             "что бот состоит в этом чате и попробуйте еще раз"
         )
+        return asyncio.create_task(
+            cleanup_command_dialog(
+                bot=bot,
+                bot_message=bot_reply,
+                delete_bot_reply=True,
+                delay=config.time_to_remove_temp_messages,
+            )
+        )
```

Comment:
Do we really need to clean up commands in private chats? I think it only makes sense in groups.

Reply:
Nice catch, I'll remove this cleanup